lib/ansible/modules/storage/ibm/ibm_sa_vol.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+#
+# Copyright (C) 2018 IBM CORPORATION
+# Author(s): Tzur Eliyahu <tzure@il.ibm.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: ibm_sa_vol
+short_description: Handle volumes on an IBM Spectrum Accelerate storage array.
+version_added: "2.7"
+
+description:
+    - "This module creates or deletes volumes to be used on
+        IBM Spectrum Accelerate storage systems"
+
+options:
+    vol:
```

Comment:
Can you add aliases like 'volume_name' as well ? same for 'pool' like 'pool_name'

Reply:
yes, I'll add.